llvm/unittests/Target/AArch64/AArch64SelectionDAGTest.cpp

```diff
@@ -1,12 +1,11 @@
-//===- llvm/unittest/CodeGen/AArch64SelectionDAGTest.cpp -------------------------===//
-//
+//===----------------------------------------------------------------------===//
```

Comment:
Not sure if it's a requirement of the license but my spot check tells me we always have the file name as part of the comment.

Reply:
IIUC it used to be a requirement; however, the coding guidelines have since removed it: https://llvm.org/docs/CodingStandards.html#file-headers